### PR TITLE
Add emoji convert

### LINKED ISSUE #469: Add emoji util
I discovered that there is still no tool to manage emoji with Thai language (convert emoji to text or convert text to emoji). I think we might add to `pythainlp.util`.

### PR DESCRIPTION
**Pull request by ppirch**
### What does this changes

Implement emoji convert for Thai language from https://www.emojiall.com/th/all-emojis

Will close #469

### CODE REVIEW

**Comment by wannaphong** [comment]
Why you close this pull request?

**Comment by ppirch** [comment]
> Why you close this pull request?

I forgot import function in unit test

**Comment by ppirch** [comment]
> > Why you close this pull request?
> 
> I forgot import function in unit test

I'm finished new commit. You can check it.

File: pythainlp/util/emojiconv.py [hunk 1]
```diff
@@ -0,0 +1,1855 @@
...
+        # output: จะมานั่งรถเมล์เหมือนผมก็ได้นะครับ ใกล้ชิดประชาชนดี :หน้ายิ้มยิงฟัน:
+
+        emoji_to_thai("หิวข้าวอยากกินอาหารญี่ปุ่น 🍣")
+        # output: หิวข้าวอยากกินอาหารญี่ปุ่น :ซูชิ:
+
+        emoji_to_thai("🇹🇭 นี่คิือธงประเทศไทย")
+        # output: :ธง_ไทย: นี่คิือธงประเทศไทย
+    """
+
+    emoji_regex = re.compile("|".join(map(re.escape, _emojis)))
```
Comment (bact) [inline pythainlp/util/emojiconv.py]: Maybe we can compile this in advanced, outside of the function.

**Comment by wannaphong** [comment]
Google colab for test https://colab.research.google.com/drive/1rPhv0vRk1a3vCu1oWz9-ehs3FmLDbkqj?usp=sharing

File: pythainlp/util/emojiconv.py [hunk 2]
```diff
@@ -0,0 +1,1855 @@
...
+
+_emojis = sorted(_emoji_th.keys(), key=len, reverse=True)
+_delimiter = ":"
+
+
+def emoji_to_thai(text: str, delimiters=(_delimiter, _delimiter)) -> str:
+    """
+    This function convert emoji to thai meaning
+
+    :param str text_Text with Emoji
```
Comment (wannaphong) [inline pythainlp/util/emojiconv.py]: `:param str text_Text with Emoji` -> `:param str text: Text with Emoji `

File: pythainlp/util/emojiconv.py [hunk 3]
```diff
@@ -0,0 +1,1855 @@
...
+    :return_Text with Emoji being converted to thai meaning
```
Comment (wannaphong) [inline pythainlp/util/emojiconv.py]: `:return_Text with Emoji being converted to thai meaning` -> `:return: Text with Emoji being converted to thai meaning`

File: pythainlp/util/emojiconv.py [hunk 4]
```diff
@@ -0,0 +1,1855 @@
...
+    :rtype_str
```
Comment (wannaphong) [inline pythainlp/util/emojiconv.py]: `:rtype_str` -> `:rtype: str`

File: pythainlp/util/emojiconv.py [hunk 5]
```diff
@@ -0,0 +1,1855 @@
...
+    
+    :Example:
+     ::
```
Comment (wannaphong) [inline pythainlp/util/emojiconv.py]: delete one indent

**Review by wannaphong — COMMENTED** [review]
Thank you for pull request. Your code looks good. I review some code. PyThaiNLP use [docstring reStructuredText style](https://gist.github.com/jesuGMZ/d83b5e9de7ccc16f71c02adf7d2f3f44). I suggest you modify the document format according to my comments.

**Comment by ppirch** [comment]
> Thank you for pull request. Your code looks good. I review some code. PyThaiNLP use [docstring reStructuredText style](https://gist.github.com/jesuGMZ/d83b5e9de7ccc16f71c02adf7d2f3f44). I suggest you modify the document format according to my comments.

Thank for reviewing my code. I update document format based on your suggestions.

**Comment by wannaphong** [comment]
Happy New Year!

**Review by wannaphong — APPROVED** [review]
I thinks it's pass. wait @bact

**Comment by ppirch** [comment]
> Happy New Year!

Happy New Year 2021 ^_^

**Review by bact — APPROVED** [review]
Thanks for the contribution. Looks good.

File: pythainlp/util/emojiconv.py (hunk 1 above)
Comment (ppirch) [reply pythainlp/util/emojiconv.py]: Wait a sec.

Comment (ppirch) [reply pythainlp/util/emojiconv.py]: It has been done.

**Pull request merged**